packages/lib/src/formatters/formatters.test.ts

```diff
@@ -0,0 +1,30 @@
+import { describe, test } from 'node:test';
+import { equal } from 'node:assert/strict';
+import { formatConfigs } from '.';
+
+describe('Formatters', () => {
+  const json = {
+    KEY0: 'KEY0',
+    KEY1: 'KEY1',
+  };
+
+  test('should export to dotenv format', () => {
+    const formatted = formatConfigs({ json, format: 'Dotenv', label: 'test' });
+    equal(formatted, `KEY0=KEY0\nKEY1=KEY1`);
+  });
+
+  test('should export to ini format', () => {
+    const formatted = formatConfigs({ json, format: 'INI', label: 'test' });
+    equal(formatted, `KEY0=KEY0\nKEY1=KEY1\n`);
+  });
+
+  test('should export to yaml format', () => {
+    const formatted = formatConfigs({ json, format: 'YAML', label: 'test' });
+    equal(formatted, 'KEY0: KEY0\nKEY1: KEY1\n');
+  });
+
+  test('should export to json format', () => {
+    const formatted = formatConfigs({ json, format: 'JSON', label: 'test' });
+    equal(formatted, '{\n  "KEY0": "KEY0",\n  "KEY1": "KEY1"\n}');
+  });
+});
```

Comment:
I made some tests for basic formats.
I decided to put tests inside `lib` dir where tested feature "lives" - I see most tests are located inside `ts` dir though. 

I don't see any existing test cases for other formatters, so I guess I was supposed to create them?

Reply:
@michaldziuba03 Thanks for adding tests for the new formatter! `Configu` has been lacking in this area, and we plan to invest heavily in drastically increasing our test coverage.